### PR TITLE
ctffind_runner.ccp: Support spaces in file names

### PR DESCRIPTION
**Pull request by GydoVanZundert**

### CODE REVIEW

File: src/ctffind_runner.cpp [hunk 1]
```diff
@@ -713,7 +713,7 @@ void CtffindRunner::executeCtffind4(long int imic)
 
 	// Write script to run ctffind
 	fh << "#!/usr/bin/env " << fn_shell << std::endl;
-	fh << fn_ctffind_exe << ctffind4_options << " > " << fn_log << " << EOF"<<std::endl;
+	fh << fn_ctffind_exe << ctffind4_options << " > '" << fn_log << "' << EOF"<<std::endl;
```
Comment (Poko18) [inline src/ctffind_runner.cpp:745]: ver5.0-cryocloud branch also has support for Ctffind5, so I guess these changes should also be applied under `void CtffindRunner::executeCtffind5` and `CtffindRunner::getCtffind5Results`

Comment (GydoVanZundert) [reply src/ctffind_runner.cpp:745]: Rebased on ver5.0-cryocloud and updated also commands for ctffind5

**Review by Poko18 — APPROVED** [review]
Not exactly sure what was the problem here `fn_log = line.substr(46);`, otherwise looks good

**Comment by GydoVanZundert** [comment]
`fn_log = words[... -1]` will not take the whole file name, instead only the last part after the last space (` `), so an incomplete filename. Now it will take the whole filename starting from column 46, which I assume is a fixed spacing in CTFFIND4/5

**Comment by Poko18** [comment]
I understand. And we are sure that column 46 is fixed spacing for both, right? As I recall, CTFFIND4 and 5 were quite simillar :)

**Comment by GydoVanZundert** [comment]
I have made several observations where it was indeed column 46 for CTFFIND4 :) I'm now checking 5.

Edit: CTFFIND5 also provides the summary file name starting in column 46

**Pull request merged**